<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Model</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Model Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="mbwNewModelName">
             <property name="text">
              <string>NewModel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Parameters</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="3" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Well</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_8">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item row="2" column="0" colspan="3">
                  <widget class="QTableWidget" name="mbwWellTableWidget">
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>100</number>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>Well name</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>X</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Y</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Weight*</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Inverse Distance Exponent*</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="mbwAddWellButton">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="mbwRemoveWellButton">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="3">
                  <widget class="QLineEdit" name="mbwWellPath">
                   <property name="text">
                    <string>Add the Well file path</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Geometry models</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_7">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item row="2" column="0" colspan="3">
                  <widget class="QTableWidget" name="mbwGeometryTableWidget">
                   <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>200</number>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>Seismic data</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>IL</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>XL</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="mbwAddGeometryButton">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="mbwRemoveGeometryButton">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="0" colspan="3">
                  <widget class="QLineEdit" name="mbwGeometryPath">
                   <property name="text">
                    <string>Add the Geometry file path</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Layers and Horizons</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="1" column="2">
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="2" column="0" colspan="3">
                  <widget class="QTableWidget" name="mbwHorizonTableWidget">
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>200</number>
                   </attribute>
                   <attribute name="horizontalHeaderMinimumSectionSize">
                    <number>50</number>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>Horizons</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Top</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Bottom</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="mbwAddHorizonButton">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="mbwRemoveHorizonButton">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="3">
                  <widget class="QLineEdit" name="mbwHorizonPath">
                   <property name="text">
                    <string>Add the Horizons file path</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Output</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_14">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_13">
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="3">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Horizontal Hz Cut</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLineEdit" name="mbwLowPassFilterVerHzCut">
                   <property name="text">
                    <string>30</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Vertical Hz Cut</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="mbwLowPassFilterCheckbox">
                   <property name="text">
                    <string>Apply a low pass filter with</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QLineEdit" name="mbwLowPassFilterHorHzCut">
                   <property name="text">
                    <string>15</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_15">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="mbwCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="mbwConfirmButton">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
